content/posts/irods-development-update-july-2025.md

```diff
@@ -0,0 +1,259 @@
+Title: iRODS Development Update: July 2025
+Date: 2025-07-25 12:00
+Author: Kory Draughn
+Slug: irods-development-update-july-2025
+Status: published
+
+
+Hello reader,
+
+After a much needed break, the iRODS team is back and ready to start the next development cycle.
+
+For the iCommands, `irsync` has been updated to not error out when stale replicas are skipped. Work to make `irsync -l` not create collections unexpectedly is in development as well.
+
+On the build and packaging side of things, support for Enterprise Linux 10 is in progress. As always, we're working to improve management of externals packages.
+
+The S3 resource plugin has received some important updates. The plugin no longer crashes when writing large files in cache mode for iRODS 5. Decoupled mode is now honored following a server redirect. We've fixed a couple upload bugs related to parallel transfer over the zone port.
```

Comment:
what does the 'zone port' part mean?

Reply:
It's port 1247. I didn't want to assume that every deployment ran on that port.

Reply:
makes sense... resolving.